change/@fluentui-react-796926b5-641c-4e20-b1c6-1577e5a5411c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
Except for react-conformance itself, all of these should be `"type": "none"` (and remove the `dependentChangeType` line) since react-conformance is a dev dependency which doesn't affect the published package.